dotCMS/src/main/webapp/html/portlet/ext/cmsconfig/company.jsp

```diff
@@ -276,3 +320,28 @@
         <%= LanguageUtil.get(pageContext, "save") %>
     </button>
 </div>
+
+<div id="regenerateKeyDialog" dojoType="dijit.Dialog" style="display:none;width:500px;vertical-align: middle; " draggable="true" title="<%= LanguageUtil.get(pageContext, "key-digest-regenerate-prompt") %>" >
+
+	<span class="ui-confirmdialog-message" >
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5677177)